browser-extension/src/utils.js

```diff
@@ -103,3 +103,104 @@ export const getRandomSubarray = (arr, size) => {
 export const getVideoStatistics = (videoId) => {
   return fetchTournesolApi(`videos/?video_id=${videoId}`, 'GET', {});
 };
+
+export const millifyViews = (videoViews) => {
+  videoViews = videoViews.toString().replace(/[^0-9.]/g, '');
+  if (videoViews < 1000) {
+    return videoViews;
+  }
+  let si = [
+    { v: 1e3, s: 'K' },
+    { v: 1e6, s: 'M' },
+    { v: 1e9, s: 'B' },
+  ];
+  let index;
+  for (index = si.length - 1; index > 0; index--) {
+    if (videoViews >= si[index].v) {
+      break;
+    }
+  }
```

Comment:
A simpler manner to write this method which also makes it in my opinion a lot more human readable (this is important for projects where a lot of people collaborate over a long time):

```
const thresholds = [[1e9, 'B'], [1e6, 'M'], [1e3, 'K'], [1, '']]
const [divider, unit] = thresholds.find(([d,_]) => videoViews >= d)
const simplifiedViewCount = (videoViews / divider)
      .toFixed(1)
      .replace(/\.0+$|(\.[0-9]*[1-9])0+$/, '$1')
return `${simplifiedViewCount}${unit}`
```

<img width="463" alt="Screenshot 2023-02-11 at 07 57 28" src="https://user-images.githubusercontent.com/8818081/218245287-fad088c6-69f2-42c3-af8c-53aa8bbf2f6a.png">

<img width="477" alt="Screenshot 2023-02-11 at 07 58 07" src="https://user-images.githubusercontent.com/8818081/218245298-9f105fa2-7ef4-4537-b289-ee8f9050191f.png">

Reply:
I'll try to make it cleaner and more readable for long-term collaboration teams. Thanks for the feedback!